.github/workflows/check.yml

```
@@ -0,0 +1,38 @@
+name: Check
+
+on:
+  push:
+    branches: master
+  pull_request:
+
+env:
+  CRYPTKEEPER_DEBUG: false
+  INFURA_API_KEY: ${{ secrets.INFURA_API_KEY }}
+  ALCHEMY_API_KEY: ${{ secrets.ALCHEMY_API_KEY }}
+  FREIGHT_TRUST_NETWORK: ${{ secrets.FREIGHT_TRUST_NETWORK }}
+  PULSECHAIN_API_KEY: ${{ secrets.PULSECHAIN_API_KEY }}
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [16, 18]
+        command: [prettier, lint, types, tests:coverage]
```

```suggestion
        command: ["prettier", "lint", "types", "tests:coverage"]
```

`:` needs to be wrapped by quotes to avoid following YAML syntax error

```
[Invalid workflow file: .github/workflows/check.yml#L1](https://github.com/CryptKeeperZK/crypt-keeper-extension/actions/runs/4366355170/workflow)
The workflow is not valid. .github/workflows/check.yml: (Line: 26, Col: 42, Idx: 589) - (Line: 26, Col: 47, Idx: 594): While scanning a plain scalar, find unexpected ':'.
```
(see: https://github.com/CryptKeeperZK/crypt-keeper-extension/actions/runs/4366355170)